appveyor.yml

```diff
@@ -0,0 +1,31 @@
+# General Configuration
+version: '{build}'
+
+# Environment Configuration
+image: Visual Studio 2017
+cache:
+  - '%LocalAppData%\NuGet\v3-cache'
+
+install:
+  #init and update submodules
+  - git submodule update --recursive --init
+
+
+# Build Configuration
+platform: Any CPU
+configuration: Release
+
+before_build:
+  - nuget restore
+
+build:
+  project: SpeckleDynamo.sln
+  verbosity: minimal
+
+after_build:
+  # zip everything into a zip file containing package folder structure
+  - 7z a SpeckleDynamo-$(appveyor_build_version).zip %APPVEYOR_BUILD_FOLDER%\SpeckleDynamo\bin\Release\SpeckleDynamo\*
```

Comment:
@alvpickmans Just spotted this... `$(appveyor_build_version)` should be `%APPVEYOR_BUILD_VERSION%`

Reply:
I think this will fix the missing build artifacts

Reply:
you areprobably right, I will update this now and if fails I will just leave it as SpeckleDynamo.zip :)